gui/gui/src/TGFont.cxx

```diff
@@ -844,19 +844,22 @@ TGTextLayout *TGFont::ComputeTextLayout(const char *string, Int_t numChars,
    chunk = layout->fChunks;
    if (chunk) y = chunk->fY;
    for (n = 0; n < layout->fNumChunks; n++) {
-      int extra;
+      int extra = maxWidth;
 
       if (chunk->fY != y) {
          curLine++;
          y = chunk->fY;
       }
-      extra = maxWidth - lineLengths[curLine];
+      if (curLine < maxLines)
+         extra = maxWidth - lineLengths[curLine];
       if (justify == kTextCenterX) {
          chunk->fX += extra / 2;
       } else if (justify == kTextRight) {
          chunk->fX += extra;
       }
       ++chunk;
+      if (!chunk)
+         break;
    }
```

Comment:
This check us useless here, one need to prevent that loop execution enter when there is no `chunk`.
By incrementing chunk it will never be a 0